Source/Scene/PostProcessLibrary.js

```diff
@@ -123,6 +129,45 @@ define([
             get : function() {
                 return createAmbientOcclusionStage();
             }
+        },
+
+        /**
+         * private
+         */
```

Comment:
Can you write descriptions for all the new stages - they don't need to be private.